scripts/dev_scripts/integration_tests.sh

```diff
@@ -398,6 +398,7 @@ run_macaron_clean $ANALYZE -purl pkg:maven/io.github.behnazh-w.demo/example-mave
 check_or_update_expected_output $COMPARE_DEPS $DEP_RESULT $DEP_EXPECTED || log_fail
 
 python3 ./tests/integration/run.py run \
+    --exclude-tag token_file_clean_up \
```

Comment:
I am thinking instead of changing this tag to `docker-only` to make it more generic :thinking: Please let me know what you think about it.

Reply:
Do you mean to rename the test tag from `token_file_clean_up` to `docker-only`?

Reply:
Yes, that is correct. I think that if we rename it to `docker-only`, in the future we don't need to add a new tag if we add any new docker-specific test.

Reply:
Thanks, sounds good to me.

Reply:
Addressed in https://github.com/oracle/macaron/pull/780/commits/bb3d48f74110cb6cd1927af9e24bb5f89a61dc98 and https://github.com/oracle/macaron/pull/780/commits/ace58a760f964f348c35130bcb27e66ca7589a0d